src/search/containers/CurrentSearchPage.js

```diff
@@ -0,0 +1,35 @@
+import { connect } from 'react-redux';
+import SearchPage from '../components/SearchPage';
+import { setQuery } from '../actions';
+
+function getVisibleAddons(addons, query) {
+  const matches = [];
+  for (const slug in addons) {
+    if (addons[slug].title.indexOf(query) >= 0) {
```

Comment:
I suppose an alternative would be to get fancy and use `Array.prototype.includes` but you'd need a polyfill for full support: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes